backend/curation/api/v1/curation/collections/common.py

```diff
@@ -198,6 +208,20 @@ def reshape_dataset_for_curation_api(dataset_version: DatasetVersion, use_canoni
     return ds
 
 
+def reshape_dataset_for_curation_api(dataset_version: DatasetVersion, use_canonical_url: bool, preview=False) -> dict:
+    dataset = reshape_dataset_for_curation_api_common(dataset_version, use_canonical_url, preview)
+    if not preview:
+        dataset["published_at"] = dataset_version.canonical_dataset.published_at
+        dataset["revised_at"] = dataset_version.canonical_dataset.revised_at
+    return dataset
+
+
+def reshape_dataset_for_curation_api_as_version(
```

Comment:
I don't really see the value of having this separate util function if it just calls another function with the same args

Reply:
If the only diff is the IF statement in reshape_dataset_for_curation_api, I'd rather just consolidate all of it into the previous singular  reshape_dataset_for_curation_api function and handle the conditional in there

Reply:
all of it == all 3 of reshape_dataset_for_curation_api_as_version, reshape_dataset_for_curation_api, and reshape_dataset_for_curation_api_common